flatland_plugins/src/world_modifier.cpp

```diff
@@ -68,6 +68,18 @@ using namespace flatland_server;
 
 namespace flatland_plugins {
 
+float RayTrace::ReportFixture(b2Fixture *fixture, const b2Vec2 &point,
+                              const b2Vec2 &normal, float fraction) {
+  // only register hit in the specified layers
+  if (!(fixture->GetFilterData().categoryBits & 1)) {
```

Comment:
Does this mean "only the first layer"? If so, that's not really the "specified" layers?